conans/client/conf/detect.py

```diff
@@ -265,6 +265,11 @@ def _detect_compiler_version(result, output, profile_path):
             result.append(("compiler.base.version", "4.8"))
         else:
             result.append(("compiler.base.version", "4.4"))
+    elif compiler == "msvc":
+        # Add default mandatory fields for MSVC compiler
+        result.append(("compiler.cppstd", "14"))
+        result.append(("compiler.runtime", "static"))
```

Comment:
```suggestion
        result.append(("compiler.runtime", "dynamic"))
```

I personally would put `dynamic` as a default for a few reasons:
- in Visual Studio itself, `MultiThreadedDll` is a default setting
- for the old "Visual Studio" compiler we detect MD as a default runtime
- I think dynamic CRT is recommended choice by Microsoft for security reasons